x/staking/client/cli/query.go

```diff
@@ -909,6 +918,7 @@ $ %s query staking all-tokenize-share-records
 	}
 
 	flags.AddQueryFlagsToCmd(cmd)
+	flags.AddPaginationFlagsToCmd(cmd, "delegations")
```

Comment:
change flag name for tokenize share records (I'm not sure what it's supposed to be)